README.md

```diff
@@ -431,6 +431,7 @@ Feel free to propose your own bot to our collection!
  - [magnickolas/remindee-bot](https://github.com/magnickolas/remindee-bot) -- Telegram bot for managing reminders.
  - [cyberknight777/knight-bot](https://gitlab.com/cyberknight777/knight-bot) -- A Telegram bot with variety of fun features.
  - [wa7sa34cx/the-black-box-bot](https://github.com/wa7sa34cx/the-black-box-bot) -- This is the Black Box Telegram bot. You can hold any items in it.
+ - [crapstone/hsctt](https://codeberg.org/crapstones-bots/hsctt/src/branch/main/telegram) -- A Telegram bot that searches for HTTP status codes in all messages and replies with the text form.
```

Comment:
```suggestion
 - [crapstone/hsctt](https://codeberg.org/crapstones-bots/hsctt) -- A Telegram bot that searches for HTTP status codes in all messages and replies with the text form.
```

It looks like the link you provided doesn't show README.

Reply:
Oh you're right. I tripped over that weird link when I pasted it, but thought that it is a Gitea quirk.